plugins/doc_fragments/aci.py

```diff
@@ -104,6 +104,15 @@ class ModuleDocFragment(object):
     - The current object including default values will be unverifiable in a single task.
     type: bool
     aliases: [ no_verification, no_verify, suppress_verify ]
+  suppress_previous:
+    description:
+    - If C(true), a GET to check previous will not be sent before a POST update to APIC.
+    - If the value is not specified in the task, the value of environment variable C(ACI_SUPPRESS_PREVIOUS) will be used instead.
+    - The default value is C(false).
+    - WARNING - This causes the previous return value to be empty.
+    - The previous state of the object will not be checked and the POST update will contain all properties.
```

Comment:
@anvitha-jain
The warning indicates that the previous configuration will not be checked when suppress_previous is enabled.
This causes the sent config in the POST to contain all options/properties specified in the task instead of just the changed properties.